os/board/esp32_devkit/Kconfig

```diff
@@ -232,3 +232,18 @@ config ESP32_AUTOMOUNT_USERFS_MOUNTPOINT
     ---help---
         Specifies the mount point where user r/w file system
         will be mounted at.
+
+config ESP32_AUTOMOUNT_ROMFS_DEVNAME
+    string "Device name of the partition for rom read only file system"
+    default "/dev/mtdblock1"
```

Comment:
is this device name should obey some rule from the partition arrange and also the source code logic ? if so, better add guide of naming in below help

Reply:
okay, improved